incubator/hnc/hack/post-submit-test.sh

```diff
@@ -0,0 +1,23 @@
+# This file is run by Prow during all postsubmits
+export PATH=$(go env GOPATH)/bin:$PATH
+mkdir -p $(go env GOPATH)/bin
+
+echo "Installing 'kubebuilder' to include the Ginkgo test suite requirements"
```

Comment:
Are you sure we need this? None of our test scripts use kubebuilder, and Ginkgo is included in our vendors/ directory.